### DISCUSSION TITLE
Why Bun is faster than Node JS?

### DISCUSSION THREAD

**Discussion by dawit-melka** [community/community#67620] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Lets discuss on Bun vs Node JS

**Comment by dawit-melka** [comment]
Bun is fast because it uses the JavaScriptCore engine, while Node.js uses the JavaScript V8 engine. The former has been optimized for faster startup time. If you want to get things done faster, you should consider replacing Node.js with Bun.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
